requirements-dev.txt

```diff
@@ -11,3 +11,5 @@ transifex-client==0.12.2
 unittest2==1.1.0
 zope.testrunner==4.5.1
 selenium==3.141.0
+linuxdoc @ git+http://github.com/return42/linuxdoc.git
```

Comment:
It would be better to depend on a release rather than the last commit.

Reply:
As long, as we do not have patched releases, searx itself is always updated to last commit on sear:/master.

I understand your concerns and if you want, I can changed the requirement to a fixed one.  But let me also hint, that I am the maintainer of linuxdoc.  We also do not maintain releases at linuxdoc and recommend to regularly update instances to topmost commit master.

Should I changed to a fixed version or is it OK for you to go with the linuxdoc master branch?